libs/ledger/tests/chaincode/smart_contract_tests.cpp

```diff
@@ -567,4 +570,357 @@ TEST_F(SmartContractTests, CheckShardedStateSetWithAddressAsName)
   VerifyQuery("query_foo", int32_t{20}, request);
 }
 
+bool Replace(std::string &orig, std::string const &what, std::string const &with)
```

Comment:
This probably can be made a library function, if there does not exist one already.